<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseViewClass</class>
 <widget class="QWidget" name="CaseViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaseView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Goal">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Goal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblGoalName">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Goal Name</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QTableWidget" name="tblParameters">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="txtCaseName"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Game State">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Game State</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTableWidget" name="tblGameState">
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::NoPen</enum>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Performance">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>382</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Performance</string>
        </attribute>
        <widget class="QLabel" name="lblSuccessCount">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>91</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Success Count</string>
         </property>
        </widget>
        <widget class="QLabel" name="lblTrialCount">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>81</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>Trial Count</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtSuccessCount">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="inputMask">
          <string>9999999999</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtTrialCount">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="inputMask">
          <string>9999999999</string>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="planViewLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
